src/main.c

```diff
@@ -271,6 +285,7 @@ int main(int argc, char* argv[])
         { "chatmix", no_argument, NULL, 'm' },
         { "dev", no_argument, NULL, 0 },
         { "help", no_argument, NULL, 'h' },
+        { "equalizer-preset", required_argument, NULL, 'e' },
```

Comment:
@Sapd I am also wondering about the shortcut for this. Maybe it should'be just `p`? I could see `e` be used for custom preset support.

Reply:
Yep good suggestion 

Rest looks good

Reply:
Updated!